src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVKmerizer.java

```diff
@@ -41,26 +56,25 @@ public boolean hasNext() {
     public SVKmer next() {
         if ( nextKmer == null ) throw new NoSuchElementException("Kmerization sequence exhausted.");
         final SVKmer result = nextKmer;
-        nextKmer = nextKmer(nextKmer, kSize-1);
+        nextKmer = nextKmer(nextKmer, kSize-kSpace);
```

Comment:
This is the only place you use kSpace.  (This is very slick, by the way.  Love the economy of it.)
So you might as well precompute this difference in the constructor.  (I.e., have a field called kAdvance, rather than kSpace, and initialize it to ```kSize-kSpace``` in the constructor.)
Tiny performance improvement, but this is likely to be called billions of times.